buildSrc/src/main/java/xyz/wagyourtail/doclet/pydoclet/parsers/ClassParser.java

```diff
@@ -0,0 +1,353 @@
+                sb.append(", ").append(parameter.getSimpleName()).append(": ").append(getTypeMirrorName(parameter.asType(), false));
+            });
+            sb.append(") -> ");
+            //Main.reporter.print(Diagnostic.Kind.NOTE, getTypeMirrorName(method.getReturnType(), false) + "");
+            if (method.getReceiverType() != null) sb.append(getTypeMirrorName(method.getReturnType(), false));
+            sb.append(":\n");
+            sb.append(getTabs(2)).append("pass\n\n");
+        });
+
+        return sb.toString();
+    }
+
+    private String getImports(){
+        StringBuilder sb = new StringBuilder();
+
+        List<String> imp = new LinkedList<>();
+        //Main.reporter.print(Diagnostic.Kind.NOTE, type + ": " + importList);
+
+        imports.forEach(t -> {
+            if(!types.containsKey(t)){
+                if ((t + "").startsWith("xyz")){
+                    if(Main.typeUtils.asElement(t) != null) {
+                        String path = getClassName((TypeElement) Main.typeUtils.asElement(t)); //getPackage((TypeElement) Main.typeUtils.asElement(t)) + "." +
+                        if (!imp.contains(path)) imp.add(path);
+                    }
+                }else if((t + "").startsWith("net") || (t + "").startsWith("com") || (t + "").startsWith("io") || (t + "").startsWith("java.util") ||
+                        (t + "").startsWith("java.lang.Runnable")  || (t + "").startsWith("java.lang.Thread") || (t + "").startsWith("java.lang.Throwable") ||
+                        (t + "").startsWith("java.util.function") || (t + "").startsWith("java.lang.ref") || (t + "").startsWith("java.io") || (t + "").startsWith("org") || (t + "").startsWith("java.lang.Iterable") ||
+                        (t + "").startsWith("java.lang.StackTraceElement")){
+                    //Main.reporter.print(Diagnostic.Kind.NOTE, typeVars + "");
+                    if(!importTypeVar)
+                        importTypeVar = true;
+                    typeVars.put(
+                            getClassName((TypeElement) Main.typeUtils.asElement(t)),
+                            t + ""
+                            );
+                }
+            }
+        });
+
+        if (importOverload)
+            sb.append("from typing import overload\n");
+        if (importList)
+            sb.append("from typing import List\n");
+        if (importTypeVar)
+            sb.append("from typing import TypeVar\n");
+        if (importAny)
+            sb.append("from typing import Any\n");
+        if (importMapping)
+            sb.append("from typing import Mapping\n");
+
+        imp.forEach(s -> {
+            sb.append("from .").append(s).append(" import *\n");
+        });
+
+        sb.append("\n");
+        for(Map.Entry<String, String> entry : typeVars.entrySet()){
+            sb.append(entry.getKey()).append(" = TypeVar[\"").append(entry.getValue().replace("<", "_").replace(">", "_").replace("?", "")).append("\"]\n");
```

Comment:
This is generating `TypeVar` for List, it shouldn't and they *should* be treated as python list...
see ClientConfigV2 as an example of it failing...

Reply:
Fixed it also there is no difference between List[str] and list[str]